### ISSUE TITLE
Menu Actions button vs. Kebab Action button

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#2310]
During office hours w/ the India team, we had a great question as to when to use either. I suggested we use the action menu button on top of the list or table if the user needs to perform batched actions. However, if the user needs to perform individual actions, should the kebab show on each row/list? While we do have some guidance on the [design system site](https://design-system.hpe.design/components/menu?q=menu), it isn't clear how it is utilized within a table context. Let's further investigate, understand and create guidance.

**Comment by ericsoderberghp** [comment]
Tied to https://github.com/grommet/hpe-design-system/issues/2166

**Comment by revanth-ms** [comment]
Use case for considerations

An item has many actions and I was thinking If I could offer flexibility by allowing an action from both places (from an item and also from batch action menu)

Screenshot:
![MultiSelect](https://user-images.githubusercontent.com/37047291/168008985-95faabab-f43f-4578-999f-541062853711.png)

**Comment by taysea** [comment]
@SeamusLeonardHPE tagging you -- can you look if this is covered by the PageHeader work you're doing?

**Comment by SeamusLeonardHPE** [comment]
Closing as pattern has been established in data collection toolbar
https://design-system.hpe.design/components/data?q=data#toolbar

**Comment by jcfilben** [comment]
What we are showing in the toolbar example doesn't seem like the correct behavior
<img width="487" alt="Screenshot 2024-09-16 at 11 30 27 AM" src="https://github.com/user-attachments/assets/70b088cb-7cc0-44c8-bbc0-9a432cb2150a">

**Comment by taysea** [comment]
@jcfilben Can you file a separate ticket for us to fix the responsive behavior of this example?

**Comment by jcfilben** [comment]
Yeah I'll file a separate ticket, also just a little confused on what the guidance is for menu action button vs kebab

**Comment by jcfilben** [comment]
Issue created https://github.com/grommet/hpe-design-system/issues/4177

**Comment by jcfilben** [comment]
Should this be re-opened? Confused on where the guidance is

**Comment by taysea** [comment]
Hmm yeah it looks like we are missing specificity of guidance on individual vs batch actions and what control should trigger which. @jcfilben Can you edit the issue description to be a bit more specific?
